onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```
@@ -3312,23 +3591,24 @@ TEST(ReductionOpTest, ReduceDimWithZero2) {
     tester.Run(expect, error_msg,
                // exclude EPs that don't handle this
                {
+                   kCoreMLExecutionProvider,
+                   kCudaExecutionProvider,
```

Why disable Cuda?

---

I do not have time to make changes to Cuda kernel so this test would fail if it is not disabled for Cuda. This is an edge case test so I think it is fine for now.

---

Should we raise an exception or a warning instead then?

---

can you add a comment here (and in the test above that also disables the CUDA EP) with a link to this issue?

---

done